style/modules/event.css

```diff
@@ -0,0 +1,106 @@
+.box .event {
+    display: block;
+    width: 500px;
+    height: 720px;
+}
+
+.event--img {
+    width: 500px;
+    height: 300px;
+    object-fit: cover;
+}
+
+.event--title {
+    padding: 0px 40px 0px;
+    color: #113255;
+    font-family: "Libre Franklin";
+    font-size: 25px;
+}
+
+.event--text {
+    font-size: 17px;
+    padding: 0px 40px 0px;
+    color: #303030;
+}
+
+.event--container {
+    display: flex;
+    align-items: center;
+}
+
+.event--dateTitle {
+    color: #113255;
+    font-family: "Libre Franklin";
+    font-size: 20px;
+    flex-grow: .5;
+    padding-left: 40px;
+}
+
+.event--date {
+    font-size: 15px;
+    flex-grow: 2;
+    color: #606060;
+}
+.event--btn {
+    flex-grow: .5;
+    padding-right: 20px;
+}
+
+.event--btn2 {
+    display: none;
+}
+
+@media only screen and (max-width: 600px) {
+    .box .event {
+        display: block;
+        width: 400px;
+        height: 500px;
+    }
+    
+    .event--img {
+        width: 400px;
+        height: 200px;
+        object-fit: cover;
```

Comment:
a lot of these are duplicated from the "main" styling (the stuff not in the media query) -- the only rules that need to go in the media query are the ones that are added or changed -- everything else is better to leave out since it means you have to change those things in two places.

for example, `font-family: "Libre Franklin"` on `.event--title` is always Franklin, so you can leave htat out of the media query.